### DISCUSSION TITLE
YAML. "List of figure" in portuguese

### DISCUSSION THREAD

**Discussion by Edilson-R** [quarto-dev/quarto-cli#1517] [Q&A] [answered]
Hello. I'm new to using Quarto and I'm having trouble translating the "List Of Figures" and "List of Tables" into Portuguese. I also don't know how to add a new list that should be called "List of Abbreviations". Can anyone help me or point me to the material to solve the problem I'm facing?
In Latex I can do it, using the Overleaf editor, but I'm not able to do the YAML in the Room.

In Latex, using the "abntex" class, the code snippet looks like this in setup:

```
\addto\captionsbrazil{
\renewcommand{\listfigurename}{List of figures} % Change name from the list of illustrations to list of figures
\renewcommand{\listadesiglasname}{List of acronyms}
\renewcommand{\list of symbolsname}{List of symbols}
\renewcommand{\appendixname}{APPENDIX}
\renewcommand{\appendicesname}{Appendix}
}
```

**Comment by mcanouil** [comment]
Portuguese, is a supported language ([`src/resources/language/_language-pt.yml`](https://github.com/quarto-dev/quarto-cli/blob/main/src/resources/language/_language-pt.yml)), see the documentation about [Document Language](https://quarto.org/docs/authoring/language.html).

As described on the same page, you can also change yourself the translation and set whatever you like, for example having the document in Portuguese, but the 'List of Figures' with a French translation:

```
lang: pt
language:
  crossref-lot-title: "Liste des Figures"
```

**Reply by cscheid** [reply]
I suspect at least part of the issue here is that @Edilson-R wants to use a different document class, namely `abntex`. @Edilson-R, are you trying to use quarto with `abntex` specifically or do you mean that the brazilian translation is also not working well for quarto's base classes?

(I'm also not sure what you mean by "YAML in the Room" - I happen to be Brazilian, in case it's easier for us to communicate in portuguese :) )

**Comment by Edilson-R** [comment, marked as answer]
I just read the final part of the Quarto documentation and found the solution here: https://quarto.org/docs/authoring/language.html 

Many thanks to all my colleagues who supported me. I will close the issue. Gracias @mcanouil, @cscheid and others.
